Include/cpython/frameobject.h

```diff
@@ -34,18 +34,14 @@ struct _frame {
     PyObject *f_locals;         /* local symbol table (any mapping) */
     PyObject **f_valuestack;    /* points after the last local */
     PyObject *f_trace;          /* Trace function */
-    int f_stackdepth;           /* Depth of value stack */
-    char f_trace_lines;         /* Emit per-line trace events? */
-    char f_trace_opcodes;       /* Emit per-opcode trace events? */
-
     /* Borrowed reference to a generator, or NULL */
     PyObject *f_gen;
-
+    int f_stackdepth;           /* Depth of value stack */
     int f_lasti;                /* Last instruction if called */
     int f_lineno;               /* Current line number. Only valid if non-zero */
-    int f_iblock;               /* index in f_blockstack */
     PyFrameState f_state;       /* What state the frame is in */
-    PyTryBlock f_blockstack[CO_MAXBLOCKS]; /* for try and loop blocks */
```

Comment:
Haven't used this for for loops since 3.8.
https://github.com/python/cpython/blob/bb3e0c240bc60fe08d332ff5955d54197f79751c/Doc/whatsnew/3.8.rst#cpython-bytecode-changes